themes/govcms/govcms_ui_kit/govcms_ui_kit.info

```diff
@@ -9,6 +9,7 @@ stylesheets[all][] = dist/css/styles.css
 
 scripts[] = vendor/owl-carousel/owl.carousel.min.js
 scripts[] = vendor/object-fit-images/ofi.browser.js
+scripts[] = vendor/jquery.validate/jquery.validate.min.js
```

Comment:
This file has not been included in the commit. This is likely because vendor folder is ignored in git. Please unignore folder, add the file, and re-ignore folder. Also when doing so, can you change the folder name to use a hyphen instead of a dot? The other folders use hyphens - it's just for consistency.